### PR TITLE
Revert #512

### LINKED ISSUE #518: Revert #512
Unfortunately it looks like we can't remove Qt4 just yet.  We need to revert #512 and corresponding upstream changes.

### PR DESCRIPTION
**Pull request by corranwebster**
This reverts #512, fixes #518.

We will want some incidental changes in #512 (like updating EDM) but they can be in a separate PR.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -5,19 +5,19 @@ environment:
 
   global:
     PYTHONUNBUFFERED: "1"
-    INSTALL_EDM_VERSION: "2.4.1"
+    INSTALL_EDM_VERSION: "2.0.0"
```
Comment (kitchoi) [inline appveyor.yml:8]: Same here.

File: .travis.yml [hunk 2]
```diff
@@ -5,14 +5,14 @@ services:
 
 env:
   global:
-    - INSTALL_EDM_VERSION=2.4.1
+    - INSTALL_EDM_VERSION=2.0.0
```
Comment (kitchoi) [inline .travis.yml:8]: We probably want to upgrade the version anyway?

Comment (corranwebster) [reply .travis.yml:8]: Yep, but separate PR.

**Comment by corranwebster** [comment]
I reinstated the restriction to wxPython < 4.1 on Mac and Linux.  Tests should hopefully pass.

**Pull request merged**